Lib/multiprocessing/context.py

```diff
@@ -254,7 +254,7 @@ def get_start_method(self, allow_none=False):
         return self._actual_context._name
 
     def get_all_start_methods(self):
-        if sys.platform == 'win32':
+        if sys.platform == 'win32' or sys.platform == 'darwin':
```

Comment:
Presumably, non-spawn start methods still exist.

Reply:
You're right! I've added a new switch case to deal only with macos